### ISSUE TITLE
Exporting 4xSID tunes can crash GTUltra 

### ISSUE THREAD

**Issue by CINJ** [jpage8580/GTUltra#48]
Version affected: 1.4.x

Symptom: 
export will crash at the step after specifying the options, SID addresses and addresses of the ZP to use.

Observations:
Seems to crash specifically when Volume Change support or Sound Effect Support is enabled. 
Crash is literally just application shutdown

Platform:
WIN32

**Comment by jpage8580** [comment]
Can you send me the sng file that you’re using?
This will be due a branch out of range in 6510

**Comment by CINJ** [comment]
Pls don't share, this is for an unreleased demo. Maybe delete this comment once you download :)

[megatune.zip](https://github.com/jpage8580/GTUltra/files/9945617/megatune.zip)

**Comment by jpage8580** [comment]
Absolutely. This is just to get things to crash. I won’t share and will
delete as soon as possible.

Can you confirm how many sids this uses? I’d expect the crash to likely
happen if saving as 3 or 4 sid only.

**Comment by jpage8580** [comment]
There will likely be some kind of debug.s file that’s been saved before the
crash. That’s the generated 6510 assembly.

**Comment by CINJ** [comment]
sorry, was coding, let me check

**Comment by CINJ** [comment]
[debug_0.zip](https://github.com/jpage8580/GTUltra/files/9945671/debug_0.zip)
See debug output

**Comment by CINJ** [comment]
The one time I managed to get export to work, the playback on Mega65 was clearly missing functionality, so I enabled sound effects and volume control to check and the crash occurs. I tried other permutations, but basically, it will always die if I try to export with any setting enabled.

**Comment by jpage8580** [comment]
Thanks again. Will take a look as soon as I can

**Comment by CINJ** [comment]
Thanks @jpage8580. Pls see if you can lend a hand before the weekend. :)

**Comment by jpage8580** [comment]
Try this one

https://www.dropbox.com/s/xbhw3uaz79sd12q/gtultra.exe?dl=0

**Comment by CINJ** [comment]
That fixed the bug!
